freeipa.spec.in

```diff
@@ -344,6 +345,7 @@ Requires: libsss_autofs
 Requires: autofs
 Requires: libnfsidmap
 Requires: nfs-utils
+Requires: python-icalendar >= 3.9.2
 Requires(post): policycoreutils
```

Comment:
There're still some changes that I would like to see in python-icalendar, namely https://github.com/collective/icalendar/pull/192 and https://github.com/collective/icalendar/pull/196 that should help #192 get pushed.
Those changes are not necessary for our cause but it would be nice to have them. Also, we'll be needing python-icalendar >= 3.10 because of https://github.com/collective/icalendar/pull/189, I will bump the version in the fixing patch.